Makd.mak

```diff
@@ -662,6 +668,17 @@ $O/doc.stamp: $(call find,$(SRC),-type f \( -name '*.d' -o -name '*.di' \))
 
 doc += $O/doc.stamp
 
+# Formatter rules
+###################
+
+# General rule to run the dfmt formatter 
+$O/dfmt.stamp: $(shell mkdir -p formatted)
+	$(shell find $(SRC) -type f \( -name '*.d' -o -name '*.di' \))
+	$(call exec,$(DFMT) $(DFMTFLAGS) $(SRC) > formatted/$(SRC) ,fmt)
```

Comment:
This doesn't make sense either :)

`$(SRC)` is a directory, so you can't redirect files to a directory.

Depending on what options `dfmt` supports, you might want to convert one file at the time instead (it might be slower though):

```make
$O/formatted/%: $(SRC)/%
    $(call exec,$(DFMT) $(DFMTFLAGS) $^ > $@,fmt)
```

This is a *pattern rule* and tell make how to build a file based on another file and a pattern. You still need to create a rule to list all the target files. But then I wonder what is the purpose of storing all these files? Is to make a diff?